### ISSUE TITLE
German Translation for the data pack?

### ISSUE THREAD

**Issue by Atten007** [ICY105/Mechanization#212]
Hello,

I know that this isn't a bug, but I'm very interested in providing a German Translation for your data pack. I'm from Germany, so I can translate the entire data pack into German if you want.

Kind regards,
Atten007 😊

**Comment by ICY105** [comment]
Of course! Translations are listed in https://github.com/ICY105/Mechanization/tree/master/translations. You can submit a PR, or send the lang file over discord if you're in the server.

I'll accept partial translations. I assume translating item names is pretty straightforward, but translating the Mechanical Manual and advancements is harder. Also, the manual is likely to change extensively in the next update so I would skip that for now anyway.

**Comment by Atten007** [comment]
I started translating the data pack into German and when the Translation is finished, I'll send you the lang file here okay?

**Comment by ICY105** [comment]
Thank you, I will include it in the next Mech release.
